helm/default-apps-gcp/templates/apps.yaml

```diff
@@ -6,6 +6,10 @@ kind: App
 metadata:
   annotations:
     chart-operator.giantswarm.io/force-helm-upgrade: "{{ .forceUpgrade }}"
+    {{- if and .dependsOn }}
```

Comment:
`and` is used if there are two conditions that needs to be true, but i only see a one so why use it?

Reply:
My bad, this was a remnant of a previous template that I din't checked. I removed it.